chef/cookbooks/hyperv/recipes/config_nova.rb

```diff
@@ -88,25 +88,3 @@
             :openstack_log => node[:openstack][:log]
            )
 end
-
-vlan_start = node[:network][:networks][:nova_fixed][:vlan]
-num_vlans = neutron_server[:neutron][:num_vlans]
-vlan_end = [vlan_start + num_vlans - 1, 4094].min
-
-template "#{node[:openstack][:config].gsub(/\\/, "/")}/neutron_hyperv_agent.conf" do
-  source "neutron_hyperv_agent.conf.erb"
-  variables(
-            :rabbit_settings => fetch_rabbitmq_settings("nova"),
-            :openstack_location => node[:openstack][:location],
-            :openstack_log => node[:openstack][:log],
-            :neutron_networking_plugin => neutron_networking_plugin,
-            :neutron_ml2_type_drivers => neutron_ml2_type_drivers,
```

Comment:
I think you can drop the definition of this variable from earlier in the recipe.